pkgs/servers/authelia/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, lib, fetchzip, autoPatchelfHook, ... }:
+
+let
+  pname = "authelia";
+  version = "4.36.2";
+in
+stdenv.mkDerivation {
+  inherit pname;
+  inherit version;
+
+  src = fetchzip {
+    url = "https://github.com/${pname}/${pname}/releases/download/v${version}/${pname}-v${version}-linux-amd64.tar.gz";
```

Comment:
At the very least, we should also support arm and arm64 binaries, seeing that they are provided by the upstream.

Reply:
I have added support for arm64, but I couldn't figure out the 32 bit version